src/main/java/millom/sandbox/mars/weather/service/JdkService.java

```diff
@@ -0,0 +1,34 @@
+package millom.sandbox.mars.weather.service;
```

Comment:
Better to move this and all JDK API related code to its own namespace. See comment below. A bit confusing to see "mars.weatehr" references in a JDK version lookup API... :-)